libs/sdk-core/src/lnurl/auth.rs

```diff
@@ -9,9 +10,35 @@ use reqwest::Url;
 use std::str::FromStr;
 use std::sync::Arc;
 
-pub type LnUrlAuthCallbackStatus = LnUrlWithdrawCallbackStatus;
+pub(crate) mod model {
```

Comment:
Does it mean that now we have duplication of this type? Also in the models.rs?

Reply:
If there isn't any difference between them can't we use one type for both? LnUrlCallbackStatus maybe?
@ok300 what do you think?

Reply:
> If there isn't any difference between them can't we use one type for both? LnUrlCallbackStatus maybe?
> @ok300 what do you think?

Done https://github.com/breez/breez-sdk/pull/150/commits/a6a804e63aecc203e2ad4159c673672efdbcf123

Reply:
@dangeross this change introduce a change in the lnurl operations return type. It unified the result into one structure: `LnUrlCallbackStatus` that has the same structure for all lnurl operations.
Would you like to review that before we are moving forward?

Reply:
@roeierez Sure thing, I'll take a look now